google/cloud/storage/internal/openssl_util.h

```diff
@@ -80,7 +81,16 @@ inline std::string UrlsafeBase64Encode(Collection const& bytes) {
 StatusOr<std::vector<std::uint8_t>> UrlsafeBase64Decode(std::string const& str);
 
 /// Compute the MD5 hash of @p payload
-std::vector<std::uint8_t> MD5Hash(std::string const& payload);
+std::vector<std::uint8_t> MD5Hash(absl::Span<char const> payload);
```

Comment:
Why did we need to introduce a `Span` overload?

Reply:
Removed.  I am thinking that these should be receiving "bytes" as input, but we should do that in a separate PR.